src/main/java/core/basesyntax/converter/DataConverterImpl.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.converter;
+
+import core.basesyntax.transaction.FruitTransaction;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    @Override
+    public List<FruitTransaction> converterToTransaction(List<String> inputData) {

```

Comment:
You are not handling the case where the input data might be incorrect. What if the data does not contain three parts after the split operation? This can lead to an ArrayIndexOutOfBoundsException. Consider adding a check for the length of the parts array after splitting the data.